cmd_format.c

```diff
@@ -169,9 +169,11 @@ int cmd_format(int argc, char *argv[])
 		case O_group:
 		case 'g':
 			dev_opts.group = optarg;
+			if(CHECK_OPT_LOCAL(dev_opts.discard)) dev_opts.discard = false;
```

Comment:
Fix coding style please - we generally stick to Linux kernel coding style

Reply:
sorry :s this is the first time I patch my own reported bugs, I'll update it.